backend/src/test/java/click/snekhome/backend/controller/IntegrationTest.java

```diff
@@ -792,8 +793,52 @@ void expectUnauthorizedWhenLoginWithBadCredentials() throws Exception {
     }
 
     @Test
+    @DirtiesContext
     void expectUnauthorizedWhenLoginWithoutUser() throws Exception {
         mockMvc.perform(MockMvcRequestBuilders.post("/api/user/login").with(csrf()))
                 .andExpect(MockMvcResultMatchers.status().isUnauthorized()).andReturn();
     }
+
+    @Test
+    @DirtiesContext
+    @WithMockUser(username = "test")
+    void expectNodesOfOwnerWhenGettingNodesByOwnerId() throws Exception {
+        Node node1 = new Node("abc", "123", "Home", 1, 100, new Coordinates(0, 0, 0), 0, 0);
+        Node node2 = new Node("def", "123", "Office", 2, 100, new Coordinates(0, 0, 0), 0, 0);
+        nodeRepo.save(node1);
+        nodeRepo.save(node2);
+        String expected = """
```

Comment:
Eventually use of objectMapper to reduce code(jsonStrings)

Reply:
Good idea. Never used it but will try it out 👌
Thanks a lot💪